src/module/iotjs_module_process.c

```diff
@@ -39,7 +39,6 @@ static iotjs_jval_t WrapEval(const char* source, size_t length, bool* throws) {
   int len1 = strlen(wrapper[1]);
 
   iotjs_string_t code = iotjs_string_create("");
-  iotjs_string_reserve(&code, len0 + length + len1);
```

Comment:
I had the same idea, but I was not sure it must be part of this patch or not. I'll do it in this PR then.